pyanaconda/display.py

```diff
@@ -106,72 +105,81 @@ def start_spice_vd_agent():
 
 # VNC
 
-def ask_vnc_question(anaconda, vnc_server, message):
+def ask_rd_question(anaconda, grd_server, message):
     """ Ask the user if TUI or GUI-over-VNC should be started.
 
     :param anaconda: instance of the Anaconda class
-    :param vnc_server: instance of the VNC server object
+    :param grd_server: instance of the GRD server object
     :param str message: a message to show to the user together
                         with the question
+    :return: if remote desktop should be used
+    :rtype: bool
     """
     App.initialize()
     loop = App.get_event_loop()
     loop.set_quit_callback(tui_quit_callback)
-    spoke = AskVNCSpoke(anaconda.ksdata, message=message)
+    spoke = AskRDSpoke(anaconda.ksdata, message=message)
```

Comment:
This class do not exists in this commit. The commit is broken and fixed by the next one.
Please fix this issue.

Reply:
Changed the commit order. :)